instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -262,7 +286,39 @@
     return constructor.create(this);
   }
 
-  SpanSuppressionStrategy getSpanSuppressionStrategy() {
+  Tracer buildTracer() {
+    TracerBuilder tracerBuilder =
+        openTelemetry.getTracerProvider().tracerBuilder(instrumentationName);
+    if (instrumentationVersion != null) {
+      tracerBuilder.setInstrumentationVersion(instrumentationVersion);
+    }
+    if (schemaUrl != null) {
+      tracerBuilder.setSchemaUrl(schemaUrl);
+    }
+    return tracerBuilder.build();
+  }
+
+  List<RequestListener> buildRequestListeners() {
```

Comment:
_maybe_ short-circuit this if `requestMetrics.isEmpty()` (only b/c it feels weird to build a Meter if we're not going to use it...)

Reply:
Good idea, done